ReactWindows/ReactNative.Net46/Views/Image/ReactImageManager.cs

```diff
@@ -274,7 +274,10 @@ private void OnImageStatusUpdate(Border view, ImageStatusEventData status)
         /// </summary>
         /// <param name="view">The image view instance.</param>
         /// <param name="source">The source URI.</param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid", Justification = "Reviewed.")]
+#pragma warning disable AvoidAsyncVoid
         private async void SetUriFromSingleSource(Border view, string source)
```

Comment:
@leonskim Yes, if there is no `await` inside then the `async` is unnecessary, I probably just forgot to remove it when I translated the file.